dracclient/utils.py

```diff
@@ -132,7 +132,9 @@ def get_all_wsman_resource_attrs(doc, resource_uri, attr_name, nullable=False):
 
 def build_return_dict(doc, resource_uri,
                       is_commit_required_value=None,
-                      is_reboot_required_value=None):
+                      is_reboot_required_value=None,
+                      commit_required_value=None,
+                      include_commit_required=False):
```

Comment:
These changes should not be made to build_return_dict.  The commit_required parameter was deprecated and removed from python-dracclient.